### ISSUE TITLE
tootctl search deploy fails with InstancesIndex

### ISSUE THREAD

**Issue by tribela** [mastodon/mastodon#26406]
### Steps to reproduce the problem

1. Run `tootctl search deploy --only instances`
2.
3.
...

### Expected behaviour

Search index would deployed successfully

### Actual behaviour

Error

### Detailed description

<details>
<summary>Stack trace</summary>

```
/opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:768:in `exec_params': PG::UndefinedColumn: ERROR:  column "id" does not exist (ActiveRecord::StatementInvalid)
LINE 1: SELECT "id" FROM "instances" WHERE "instances"."id" IN ('0.0...
               ^

	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:768:in `block (2 levels) in exec_no_cache'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/dependencies/interlock.rb:41:in `permit_concurrent_loads'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:767:in `block in exec_no_cache'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `handle_interrupt'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `block in synchronize'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `handle_interrupt'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `synchronize'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract_adapter.rb:769:in `block in log'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/sentry-rails-5.9.0/lib/sentry/rails/tracing.rb:54:in `instrument'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract_adapter.rb:760:in `log'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:766:in `exec_no_cache'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:745:in `execute_and_clear'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql/database_statements.rb:54:in `exec_query'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/database_statements.rb:560:in `select'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/database_statements.rb:66:in `select_all'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/query_cache.rb:110:in `select_all'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/relation/calculations.rb:250:in `block in pluck'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/relation.rb:962:in `skip_query_cache_if_necessary'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/relation/calculations.rb:246:in `pluck'
	from /opt/mastodon/app/lib/importer/base_importer.rb:49:in `block in clean_up!'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search/scrolling.rb:40:in `block in scroll_batches'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search/scrolling.rb:36:in `loop'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search/scrolling.rb:36:in `scroll_batches'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search.rb:61:in `method_missing'
	from /opt/mastodon/app/lib/importer/base_importer.rb:47:in `clean_up!'
	from /opt/mastodon/lib/mastodon/cli/search.rb:83:in `block in deploy'
	from /opt/mastodon/lib/mastodon/cli/search.rb:61:in `each'
	from /opt/mastodon/lib/mastodon/cli/search.rb:61:in `deploy'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/command.rb:27:in `run'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor.rb:392:in `dispatch'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/invocation.rb:116:in `invoke'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor.rb:243:in `block in subcommand'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/command.rb:27:in `run'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor.rb:392:in `dispatch'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/base.rb:485:in `start'
	from /opt/mastodon/bin/tootctl:9:in `block in <main>'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/strategy.rb:60:in `wrap'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy.rb:154:in `strategy'
	from /opt/mastodon/bin/tootctl:8:in `<main>'
/opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:768:in `exec_params': ERROR:  column "id" does not exist (PG::UndefinedColumn)
LINE 1: SELECT "id" FROM "instances" WHERE "instances"."id" IN ('0.0...
               ^

	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:768:in `block (2 levels) in exec_no_cache'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/dependencies/interlock.rb:41:in `permit_concurrent_loads'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:767:in `block in exec_no_cache'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `handle_interrupt'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `block in synchronize'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `handle_interrupt'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `synchronize'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract_adapter.rb:769:in `block in log'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activesupport-7.0.6/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/sentry-rails-5.9.0/lib/sentry/rails/tracing.rb:54:in `instrument'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract_adapter.rb:760:in `log'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:766:in `exec_no_cache'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql_adapter.rb:745:in `execute_and_clear'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/postgresql/database_statements.rb:54:in `exec_query'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/database_statements.rb:560:in `select'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/database_statements.rb:66:in `select_all'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/query_cache.rb:110:in `select_all'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/relation/calculations.rb:250:in `block in pluck'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/relation.rb:962:in `skip_query_cache_if_necessary'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/activerecord-7.0.6/lib/active_record/relation/calculations.rb:246:in `pluck'
	from /opt/mastodon/app/lib/importer/base_importer.rb:49:in `block in clean_up!'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search/scrolling.rb:40:in `block in scroll_batches'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search/scrolling.rb:36:in `loop'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search/scrolling.rb:36:in `scroll_batches'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/search.rb:61:in `method_missing'
	from /opt/mastodon/app/lib/importer/base_importer.rb:47:in `clean_up!'
	from /opt/mastodon/lib/mastodon/cli/search.rb:83:in `block in deploy'
	from /opt/mastodon/lib/mastodon/cli/search.rb:61:in `each'
	from /opt/mastodon/lib/mastodon/cli/search.rb:61:in `deploy'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/command.rb:27:in `run'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor.rb:392:in `dispatch'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/invocation.rb:116:in `invoke'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor.rb:243:in `block in subcommand'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/command.rb:27:in `run'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor.rb:392:in `dispatch'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/thor-1.2.2/lib/thor/base.rb:485:in `start'
	from /opt/mastodon/bin/tootctl:9:in `block in <main>'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy/strategy.rb:60:in `wrap'
	from /opt/mastodon/vendor/bundle/ruby/3.2.0/gems/chewy-7.3.3/lib/chewy.rb:154:in `strategy'
	from /opt/mastodon/bin/tootctl:8:in `<main>'
```
</details>

`app/models/instance.rb` Doesn't have `id` column. That causes this issue

### Mastodon instance

qdon.space

### Mastodon version

v4.1.5 - v4.2.0-beta1

### Technical details

If this is happening on your own Mastodon server, please fill out those:
- Ruby version: 3.2.2
- Node.js version: 16.20

**Comment by ClearlyClaire** [comment]
This should be fixed by #26269

**Comment by tribela** [comment]
Oh, I didn't checked that. Closing issue
